server/controllers/reports_proposed/templates/style/reports.css

```diff
@@ -39,11 +39,15 @@ tr.outline {
   border-bottom : 1px solid #9f9f9f;
 }
 
-.center > * {
+.header_left {  
```

Comment:
Traditionally, it has been considered bad practice to use underscores in CSS class names.  In fact, for old browsers, it used to break CSS ([source](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Underscores_in_class_and_ID_Names)).

[Here is a poll on CSS tricks on using hyphens (`-`), underscores (`_`) or camelCase for class names](https://css-tricks.com/new-poll-hyphens-or-dashes/).  People seem to favor either dashes (like in bootstrap) or camelCase.